src/classes/com/sun/tdk/jcov/Instr.java

```diff
@@ -644,6 +652,14 @@ private int handleEnv_(EnvHandler opts) throws EnvHandlingException {
         include_rt = opts.getValue(DSC_INCLUDE_RT);
         Utils.checkFileCanBeNull(include_rt, "JCovRT library jarfile", Utils.CheckOptions.FILE_EXISTS, Utils.CheckOptions.FILE_ISFILE, Utils.CheckOptions.FILE_CANREAD);
 
+        try {
+            String pluginClass = opts.getValue(InstrumentationOptions.DSC_INSTR_PLUGIN);
+            if(pluginClass != null && !pluginClass.isEmpty())
+                plugin = (InstrumentationPlugin) Class.forName(opts.getValue(InstrumentationOptions.DSC_INSTR_PLUGIN)).newInstance();
```

Comment:
There should be: forName().getDeclaredConstructor().newInstance()  Class<?>.newInstance() has been deprecated since 9